src/WebGL/Renderer.ts

```diff
@@ -1797,10 +1799,12 @@ export class Renderer {
       b = 0,
       color,
       intensity,
+      // @ts-ignore
```

Comment:
what is the purpose of all the ts-ignore statement?

Reply:
Just referenced the comment and addressed it here, https://github.com/3dmol/3Dmol.js/pull/712/files#r1303022588